test/videojs-contrib-hls.test.js

```diff
@@ -2687,6 +2689,108 @@ QUnit.test('populates quality levels list when available', function(assert) {
   assert.ok(this.player.tech_.hls.qualityLevels_, 'added quality levels from video with source');
 });
 
+QUnit.test('stores bandwidth and throughput in localStorage', function(assert) {
+  const origHlsOptions = videojs.options.hls;
```

Comment:
This is a big test... it might make sense (if possible) to break it down into a few smaller tests.  Tiny tests make debugging and finding issues a little bit easier.